docs/tutorial.rst

```diff
@@ -38,104 +70,94 @@ Running simulations is very straightforward in ``msprime``::
     ┃ ┃ ┃ ┃ ┏┻┓
     3 5 0 4 1 2
 
+***************
+Sequence length
+***************
 
-Here, we simulate the coalescent for a sample of size six
-with an effective population size of 1000 diploids,
-and then print out a depiction of the resulting tree.
-The ``msprime`` library uses
-`tskit <https://tskit.readthedocs.io/en/stable>`_
-to represent simulation results and
-the :func:`.simulate` function returns a
-:class:`tskit.TreeSequence` object, which provides a very
-efficient way to access the correlated trees in simulations
-involving recombination. In this example we know that
-there can only be one tree because we have not provided
-a value for ``recombination_rate``, and it
-defaults to zero.
-Therefore, we access the only tree in the
-sequence using the :meth:`~tskit.TreeSequence.first` method.
-Finally, we draw a simple depiction of the tree to the terminal
-using the :meth:`tskit.Tree.draw` method.
+Because we haven't yet specified a sequence length, our simulated
+sequence will have length 1::
 
-Genealogical trees record the lines of descent along which genomes
-have been inherited. Since diploids have two copies of each autosomal
-chromosome, diploid individuals contain two such lines of descent:
-the simulation above provides the genealogical history of only three diploids.
+	>>> ts.sequence_length
+	1
 
-Trees are represented within ``tskit`` (and therefore ``msprime``)
-in a slightly unusual way. In
-the majority of libraries dealing with trees, each node is
-represented as an object in memory and the relationship
-between nodes as pointers between these objects. In ``tskit``,
-however, nodes are *integers*.
-In the tree above, we can see that the leaves of the tree
-are labelled with 0 to 5, and all the internal nodes of the tree
-are also integers with the root of the tree being 10.
+It is sometimes more convenient to set the sequence length to be
```

Comment:
Maybe we should make this a stronger recommendation, e.g. "It is usually much easier to keep units straight if you set the sequence length to be the number of nucleotides..."